packages/transformers/js/src/visitors/process.js

```diff
@@ -16,6 +16,17 @@ export default {
           // asset.meta.env[key.value] = process.env[key.value];
         }
       }
+    } else if (types.matchesPattern(node, 'process.browser')) {
+      // Inline process.browser
+      const assignmentAncestor = ancestors.find(ancestor => {
```

Comment:
Is there a better way of doing this? It looks a bit wired to me, but I didn't find a better solution.

Reply:
The list of ancestors should be sorted, so the last element could be the assignment expression.

Reply:
So, the following should work as well?

```js
const lastAncestor = ancestors[ancestors.length - 1];
const isAssignmentExpression = Boolean(
  lastAncestor &&
    types.isAssignmentExpression(lastAncestor) &&
    lastAncestor.left === node
);
```

Unfortunately, when doing this the tests are failing. Can it be that it is sorted within the program, but not relative to the MemberExpression?

Reply:
So, the following should work as well?

```js
const lastAncestor = ancestors[ancestors.length - 1];
const isAssignmentExpression = Boolean(
  lastAncestor &&
    types.isAssignmentExpression(lastAncestor) &&
    lastAncestor.left === node
);
```

Unfortunately, when doing this the tests are failing. Can it be that it is sorted within the program, but not relative to the MemberExpression?

Reply:
`ancestors` seems to include the current node (try [here](https://astexplorer.net/#/gist/65c6dd4d9da0841ec7b79bafa28c3f92/4071d6f7523de5f27df15c51b9fd5e07d7e8205c), open the console), so you need `- 2`

Reply:
Ah, thanks for checking, that makes sense! I updated the code...